include/aabbox3d.h

```diff
@@ -22,7 +23,7 @@ class aabbox3d // : public AllocationOverrideDefault ?
 	public:
 
 		//! Default Constructor.
-		aabbox3d(): MinEdge(-1,-1,-1), MaxEdge(1,1,1) {}
+		aabbox3d(): MinEdge(numeric_limits<T>::max()), MaxEdge(numeric_limits<T>::min()) {}

```

Comment:
wrong, `numeric_limits<float>::min()` is the closest value to 0, not the biggest negative value for that you want `lowest()`